integrations/graal/native-image-extension/src/main/java/io/helidon/integrations/graal/nativeimage/extension/HelidonReflectionFeature.java

```diff
@@ -222,6 +225,17 @@ private void addJaxRsConversions(BeforeAnalysisContext context, String annotatio
         }
     }
 
+    private void processJaxRsTypes(BeforeAnalysisContext context) {
+        tracer.parsing(() -> "Looking up JAX-RS resource methods.");
+
+        new JaxRsMethodAnalyzer(context, util)
```

Comment:
Looks a bit weird not being a static method or using a local var.

Reply:
I have merged the PR even with this comment, as I do not think of it as a technical debt. The implementation was easier using an instance.